intf/intf.go

```diff
@@ -262,3 +262,8 @@ const (
 func InterfaceState(name string, state IntState) error {
 	return accessor.InterfaceState(name, state)
 }
+
+// OverrideAccessor overrides the package network accessor to custom implementation.
+func OverrideAccessor(acc NetworkAccessor) {
+	accessor = acc
```

Comment:
I'm glad that defining this as an interface that could be overridden was useful.